src/main/scala/nl.knaw.dans.easy.bagindex/components/IndexBagStoreDatabaseComponent.scala

```diff
@@ -89,7 +89,7 @@ trait IndexBagStoreDatabaseComponent extends DatabaseComponent with DebugEnhance
       resultSet
         .map(result => Stream.continually(result.next())
           .takeWhile(b => b)
-          .map(_ => (UUID.fromString(result.getString("bagId")), DateTime.parse(result.getString("created"), dateTimeFormatter)))
+          .map(_ => (UUID.fromString(result.getString("bagId")), DateTime.parse(result.getString("created").trim, dateTimeFormatter)))
```

Comment:
I had to make this change (add `.trim`) to make Travis happy. I'm not sure why it is needed, but I discovered that it was already added to other instances of `result.getString("created")` in https://github.com/DANS-KNAW/easy-bag-index/pull/7/files#diff-c5e31c23969e50b9949b68f0261c0a04R105.

I have no idea why our database returns its result with trailing whitespace on Travis, while it doesn't on my local machine.

@janvanmansum do you remember why you added the `.trim` in other places in https://github.com/DANS-KNAW/easy-bag-index/pull/7?